includes/CLI/Plugin_Check_Command.php

```diff
@@ -124,6 +126,7 @@ public function check( $args, $assoc_args ) {
 
 		$checks_to_run = array();
 		try {
+			$runner->set_experimental_flag( $options['include-experimental'] );
```

Comment:
Just double-checking, will this always be set? Have you tested that? Unsure about how WP-CLI populates these values for boolean arguments, so I'd like us to verify this doesn't cause a PHP warning if the argument is not provided.

Reply:
@felixarntz we parse the options with [`get_options`](https://github.com/10up/plugin-check/blob/b2fb12993c6bad7147f6f2466a4c0cea93963743/includes/CLI/Plugin_Check_Command.php#L209-L215) method to make sure the default of false is included. We do the same for `--ignore-errors` and `--ignore-warnings` flags too.

Reply:
Thanks @jjgrainger!